src/main/java/com/aws/greengrass/util/platforms/unix/linux/CgroupSubSystemV2.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.util.platforms.unix.linux;
+
+import com.aws.greengrass.lifecyclemanager.GreengrassService;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.Set;
+
+@SuppressFBWarnings(value = "DMI_HARDCODED_ABSOLUTE_FILENAME",
+        justification = "CgroupSubSystemV2 virtual filesystem path cannot be relative")
+public enum CgroupSubSystemV2 implements CGroupSubSystemPath {
+    Memory, CPU, Freezer, Unified;
+    private static final String CGROUP_SUBTREE_CONTROL_CONTENT = "+cpuset +cpu +io +memory +pids";
+
+    @Override
+    public String rootMountCmd() {
+        return String.format("mount -t cgroup2 none %s", CGROUP_ROOT);
+    }
+
+    @Override
+    public Path getSubsystemRootPath() {
+        return CGROUP_ROOT;
+    }
+
+    @Override
+    public Path getSubsystemGGPath() {
+        return getSubsystemRootPath().resolve(GG_NAMESPACE);
+    }
+
+    @Override
+    public Path getSubsystemComponentPath(String componentName) {
+        return getSubsystemGGPath().resolve(componentName);
+    }
```

Comment:
These implementations are all the same in both V1 and V2. These should be defaults in the interface.

Everything which is _not_ different between V1 and V2 should be in the interface or a common parent class.